libmachine/provision/suse.go

```diff
@@ -135,51 +143,27 @@ func (provisioner *SUSEProvisioner) Provision(swarmOptions swarm.Options, authOp
 		return err
 	}
 
-	if strings.ToLower(provisioner.OsReleaseInfo.ID) != "opensuse" {
-		// This is a SLE machine, enable the containers module to have access
-		// to the docker packages
-		if _, err := provisioner.SSHCommand("sudo -E SUSEConnect -p sle-module-containers/12/$(uname -m) -r ''"); err != nil {
-			return fmt.Errorf(
-				"Error while adding the 'containers' module, make sure this machine is registered either against SUSE Customer Center (SCC) or to a local Subscription Management Tool (SMT): %v",
-				err)
-		}
-	}
-
 	log.Debug("Installing base packages")
 	for _, pkg := range provisioner.Packages {
 		if err := provisioner.Package(pkg, pkgaction.Install); err != nil {
 			return err
 		}
 	}
 
-	log.Debug("Installing docker")
-	if err := provisioner.Package("docker", pkgaction.Install); err != nil {
+	log.Debug("Configuring SuSE firewall...")
+	if err := provisioner.configureFirewall(); err != nil {
```

Comment:
The previous code checked if firewall was installed, did we remove this on purpose? Or are we configuring it because if its not installed/enabled, it won't work at all?

Reply:
Added this check back in in the latest commit!

If the firewall isn't installed / enabled, the provisioning should still work.